apps/meteor/app/models/server/models/Users.js

```diff
@@ -999,6 +999,16 @@ export class Users extends Base {
 		);
 	}
 
+	findActiveFederated(options = {}) {
+		return this.find(
+			{
+				active: true,
+				federated: true,
```

Comment:
I'd say it would be better to have a new one, since as you said this one is from the oldest federation and probably will be deprecated soon. What do you think?

Reply:
I think it may be worse having two different flags for the same purpose..

Reply:
One is for the old, v1 federation, which will be deprecated. This one is for the Matrix federation.